src/main/java/com/bootcamp/demo/controller/PaymentController.java

```diff
@@ -0,0 +1,36 @@
+package com.bootcamp.demo.controller;
+
+import com.bootcamp.demo.exception.PaymentFailException;
+import com.bootcamp.demo.model.Card;
+import com.bootcamp.demo.model.Transaction;
+import com.bootcamp.demo.service.PaymentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/payment")
```

Comment:
You could add the `produces = APPLICATION_JSON_VALUE` (from `org.springframework.util.MimeTypeUtils`) parameter to this `@RequestMapping`

Reply:
Updated base branch to feature/payment-by-card instead of master. Those are not my modifications